cssselect/parser.py

```diff
@@ -614,12 +614,13 @@ def parse_series(tokens):
     Parses the arguments for :nth-child() and friends.
 
     :raises: A list of tokens
-    :returns: :``(a, b)``
+    :returns: :``(a, b, c?)``
```

Comment:
It was decided on the meeting that this breaks the function signature and so a separate function should be made with a different return value structure.